### DISCUSSION TITLE
Retrieve the deleted forked repository

### DISCUSSION THREAD

**Discussion by nickg100** [community/community#57308] [Repositories] [answered]
### Select Topic Area

Question

### Body

We have one member in the organisation. So after removing him from organisation his all forked repositories also get deleted. He has raised a PR. But now showing repository not exist and PR get closed. Please let us know how to retrieve his deleted repo.

**Comment by airtower-luna** [comment, marked as answer]
You should still be able to check out the PR branch from the target repository, using a special ref: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally#modifying-an-inactive-pull-request-locally

Then you can push it to a suitable place and make a new PR.

**Reply by nickg100** [reply]
Thanks @airtower-luna for the help. It worked!
